settings.sh

```diff
@@ -1027,7 +1027,8 @@ instantossettings() {
         ;;
     *tools)
         DEVUTILDIR=/usr/local/share/instanttools/
-        if [ ! -d "$DEVUTILDIR" ]; then
+        DEVUTILCOMMAND=/bin/ibuild
+        if [ ! -d "$DEVUTILDIR" && ! -e "$DEVUTILCOMMAND"]; then
```

Comment:
```suggestion
        if [ ! -d "$DEVUTILDIR" ] && command -v ibuild &>/dev/null; then
```
We don't want to hardcode the path, it might change or be different on other systems.

Reply:
Right.